src/patternfly/components/DualListSelector/dual-list-selector-item-count.hbs

```diff
@@ -0,0 +1,6 @@
+<span class="pf-c-dual-list-selector__item-count{{#if dual-list-selector-item-count--modifier}} {{dual-list-selector-item-count--modifier}}{{/if}}"
```

Comment:
Need some styles for this element - looks like it needs a left margin and a background color override similar to tree view

https://github.com/patternfly/patternfly/blob/2c6324d2ec18eff85b9baee3ed2ec1a0a1b9695d/src/patternfly/components/TreeView/tree-view.scss#L38